apps/sms/services/js/drafts.js

```diff
@@ -154,9 +150,15 @@
      * Stores drafts that are held in memory to local storage.
      */
     store: function() {
+      var deferred = Utils.Promise.defer();
```

Comment:
I need this promise for the case when draft is saved before activity is closed, now when I moved draft saving into ```beforeLeave``` app-in-inline-activity-mode doesn't have chance to fire "drafts-changed" event through BroadcastChannel unless we wait for the store to be completed. That is why I wait for this promise to be resolved before I call ```ActivityClient.postResult```. As side bonus - I don't see ```IndexedDB transaction aborted``` exceptions anymore :)